static/sidebar.less

```diff
@@ -197,8 +194,12 @@ img.social-icon-small {
 				// targets just the top level's subgroup to indent on mobile
 			}
 		}
+		.collection-group:first-of-type {
+			margin-top: 0;
+		}
 		.collection-group {
 			text-transform: uppercase;
+			margin-top: 8px;
```

Comment:
This doesn’t seem right to me… shouldn’t it be 10px?

I see the difference between this PR:

<img width="238" alt="Screen Shot 2019-06-17 at 4 08 34 PM" src="https://user-images.githubusercontent.com/10070176/59642535-4851ac00-911a-11e9-8b0a-a84a148e24c2.png">

…and canjs.com:

<img width="236" alt="Screen Shot 2019-06-17 at 4 08 41 PM" src="https://user-images.githubusercontent.com/10070176/59642547-51db1400-911a-11e9-8a44-1110eee937b6.png">

I also don’t see the spacing under Data Modeling fixed.

Reply:
I changed to 10px which made Data Modelling line up better:

![datamodelling](https://user-images.githubusercontent.com/381138/59646097-28c27f80-912a-11e9-9a87-487ac6b5a7d4.png)

Reply:
When I click on can-connect, the spacing between Data Modeling and Core is still wrong.

Reply:
I removed the first-of-type declaration. Looks like that fixes it. 

![spacing](https://user-images.githubusercontent.com/381138/59704288-987b4d80-91b0-11e9-9281-47fdb639fbe4.png)

Reply:
Yup, that might fix that… but now there’s too much space when nothing is selected.

Please make the adjustments and then thoroughly test opening & closing sections, clicking on different links in the hierarchy, etc.